themes/default/content/registry/packages/scaleway/installation-configuration.md

```diff
@@ -25,7 +25,7 @@ Replace the version string with your desired version.
 
 ## Setup
 
-To provision resources with the Pulumi Scaleway provider, you need to have Scaleway API credentials. [See the Scaleway documentation](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/) for details.
+To provision resources with the Pulumi Scaleway provider, you need to have Scaleway credentials. Scaleway maintains documentation on how to create API keys [here](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/)
```

Comment:
```suggestion
To provision resources with the Pulumi Scaleway provider, you need to have Scaleway credentials. [Scaleway API keys documentation](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/)
```

we should avoid here, click here, learn more link text as its not descriptive nor accessible.